iosApp/iosApp/Pages/NearbyTransit/NearbyTransitView.swift

```diff
@@ -74,11 +77,6 @@ struct NearbyTransitView: View {
                 scrollToTop()
             }
         }
-        .onChange(of: predictionsByStop) { newPredictionsByStop in
-            if let newPredictionsByStop {
-                let condensedPredictions = newPredictionsByStop.toPredictionsStreamDataResponse()
```

Comment:
While in this file, noticed a warning that this was unused - no need to be doing this extra work!